openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -832,6 +833,50 @@ Here is another example JWT not only proving possession of a private key but als
 }
 ```
 
+#### `ldp_vp` Key Proof Type {#ldp_vp-proof-type}
+
+The verifiable presentation as defined by [@VC_DATA_2.0] or [@VC_DATA] signed using Data Integrity that are used as Key Proofs MUST contain the following elements:
+
+  * in the presentation itself,
+      * `holder`: OPTIONAL. MUST be equivalent to the controller identifier (e.g. DID) for the verificationMethod identified by the `proof.verificationMethod` property.
+
+  * in the proof body, 
```

Comment:
I am not sure the structure is accurate. proof is part of a w3c verifiable presentation, right? so isn't the following more accurate?
```
When a W3C Verifiable Presentation as defined by [@VC_DATA_2.0] or [@VC_DATA] signed using Data Integrity is used as Key Proof, it MUST contain the following elements:

* holder: ..
* proof: ...
    *  domain
    * challenge
```

Reply:
https://github.com/openid/OpenID4VCI/pull/66/commits/9e31825398e426fb22da4c09c7259e0647dcb677